features/car/oneway.feature

```diff
@@ -81,3 +81,12 @@ Feature: Car - Oneway streets
         When I route I should get
             | from | to | route    |
             | a    | c  | ab,bc,bc |
+
+
+    # Reversible oneways (low frequency) vs alternating oneways (high frequency).
+    # See: https://github.com/Project-OSRM/osrm-backend/issues/2837
+    Scenario: Car - Route over alternating but not reversible oneways
+        Then routability should be
+            | highway | oneway      | forw | backw |
+            | primary | reversible  |      |       |
+            | primary | alternating | x    | x     |
```

Comment:
What should the behavior be for implied oneways with `oneway=alternating`? I would say alternating overwrites the implied oneway then, since it's an explicit tag you have to specify.